mypy/server/update.py

```diff
@@ -342,6 +342,12 @@ def update_module(self,
         self.manager.log_fine_grained('--- update single %r ---' % module)
         self.updated_modules.append(module)
 
+        # builtins and friends could potentially get triggered because
+        # of protocol stuff, but nothing good could possibly come from
+        # actually updating them.
+        if module in ('builtins', 'typing', 'mypy_extensions', 'typing_extensions'):
```

Comment:
Do we also need to add `abc` here? Also maybe you can re-use `mypy.semanal_main.core_modules` or define a new constant?